From f6c8e85d146fa979c232b887885069bc0146e7e3 Mon Sep 17 00:00:00 2001
From: Ilya Zlobintsev <ilya.zl@protonmail.com>
Date: Wed, 2 Jul 2025 20:18:12 +0300
Subject: [PATCH] fix: use dbus-run-session instead of dbus-launch for flatpak
 daemon launch

---
 flatpak/daemon.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/flatpak/daemon.sh b/flatpak/daemon.sh
index d2ad577c..bcfe0eb3 100755
--- a/flatpak/daemon.sh
+++ b/flatpak/daemon.sh
@@ -9,4 +9,4 @@ FLATPAK_USER_DIR=/home/${FLATPAK_INSTALL_USER}/.local/share/flatpak
 export LACT_DAEMON_SOCKET_PATH="/run/host/root/run/lactd.sock"
 export LACT_DAEMON_CONFIG_DIR="/run/host/root/etc/lact"
 
-exec dbus-launch $FLATBOX_BIN run --flatpak-install-path $FLATPAK_USER_DIR --app io.github.ilya_zlobintsev.LACT lact daemon
+exec dbus-run-session $FLATBOX_BIN run --flatpak-install-path $FLATPAK_USER_DIR --app io.github.ilya_zlobintsev.LACT lact daemon
-- 
2.50.0

